### ISSUE TITLE
Missing end tags

### ISSUE THREAD

**Issue by tansengming** [ntietz/rc-ics-filter#1]
Hi @ntietz thanks for building this! It looks like there's a small bug. This is output I get when I use my token

```
BEGIN:VCALENDAR
VERSION:2.0
PRODID:icalendar-ruby
CALSCALE:GREGORIAN
METHOD:PUBLISH
NAME:RC Personal Calendar
X-WR-CALNAME:RC Personal Calendar
REFRESH-INTERVAL;VALUE=DURATION:PT1M
X-PUBLISHED-TTL:PT1M
BEGIN:VTIMEZONE
TZID:America/New_York
BEGIN:DAYLIGHT
DTSTART:20220313T030000
TZOFFSETFROM:-0500
TZOFFSETTO:-0400
RRULE:FREQ=YEARLY;BYDAY=2SU;BYMONTH=3
TZNAME:EDT
END:DAYLIGHT
BEGIN:STANDARD
DTSTART:20221106T010000
TZOFFSETFROM:-0400
TZOFFSETTO:-0500
RRULE:FREQ=YEARLY;BYDAY=1SU;BYMONTH=11
TZNAME:EST
END:STANDARD
END:VTIMEZONE
BEGIN:

```

It looks like it's missing `END:VCALENDAR` ? I don't know the calendar protocol but it looks like it might need it?

**Comment by ntietz** [comment]
Ahh, it looks like this happens if the last event listed is a cancelled event. Thank you for reporting this @tansengming !

I've just pushed in a fix in de01b5e and it's deployed. Let me know if that resolves it for you!

**Comment by tansengming** [comment]
Thanks it looks like it's working now!
